CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 - `formService.getGeoscapeReverseGeocoding()`
+- **BREAKING** update `searchCivicPlusHCMSContentItems` to call HCMS directly
```

Comment:
This should not be a breaking change and should prob be under the `### Changed` heading.

Reply:
well, previous versions of apps won't work unless we bring back the endpoint in the api.

Reply:
The api changes do not dictate the semver version of this package.